core/src/main/kotlin/com/bumble/appyx/Appyx.kt

```diff
@@ -0,0 +1,19 @@
+package com.bumble.appyx
+
+import com.bumble.appyx.core.children.ChildEntry
+
+object Appyx {
+
+    var exceptionHandler: ((Exception) -> Unit)? = null
+    var defaultKeepMode: ChildEntry.KeepMode = ChildEntry.KeepMode.KEEP
```

Comment:
Make `Suspend` as default?

Reply:
The current default is `KEEP`, do not want to introduce breaking changes. I am pretty sure that we will find a lot of bugs because of it in our app.

Reply:
Yeah, you're right. We probably need to explicitly change mode to `Suspend` and analyse it in a migration task